File: OOPSeminar5/MVCExample/MVCStudentApp/src/Model/Core/Person.java
package Model.Core;

/**
 * Родительский класс
 */
public class Person {
    // Поля
    private String name;
    private int age;

    /**
     * Конструктор
     * 
     * @param name // Имя
     * @param age  // Возраст
     */
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    // Методы
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    // Переопределение метода toString
    @Override
    public String toString() {
        return String.format("Имя = %s; Возраст = %d", getName(), getAge());
    }
}

File: OOPSeminar5/python_mvc_app/model/core/person.py
class Person:
    def __init__(self, name: str, age: int):
        self.name = name
        self.age = age

    def getName(self) -> str:
        return self.name

    def setName(self, name: str) -> None:
        self.name = name

    def getAge(self) -> int:
        return self.age

    def setAge(self, age: int) -> None:
        self.age = age

    def __str__(self) -> str:
        return f"Person [name={self.name}, age={self.age}]"